### DISCUSSION TITLE
記事ページのOGP画像で最終行の下部が見切れている

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#74] [要望・アイデア・不具合]
記事ページのOGP画像には記事タイトルが含まれますが、その最終行の下部が見切れます。

![https___qiita-user-contents imgix net_https%3A%2F%2Fqiita-user-contents imgix net%2Fhttps%253A%252F%252Fcdn qiita com%252Fassets%252Fpublic%252Farticle-ogp-background-9f5428127621718a910c8b63951390ad png%3Fixlib%3Drb-4 0 0%26w%3D1200%26mark](https://user-images.githubusercontent.com/20086673/144245409-b7c067fe-ca64-4a0f-ab1e-788c0074e9b4.png)

**Comment by kskwtnk** [comment]
@munierujp 

発見いただいてありがとうございます！
すぐに原因が思い浮かばないのですが、調査いたします！

**Comment by kskwtnk** [comment]
@munierujp 

こちら修正のリリースをして、解消しました！

![](https://user-images.githubusercontent.com/22670303/144574238-d79af1be-1a7e-4e0e-8bc6-94886ff5ef34.png)

**Reply by munierujp** [reply]
@xrxoxcxox 素早い対応！ありがとうございます🙇‍♂️

**Reply by kskwtnk** [reply]
@munierujp こちらこそ、OGPアップデートの後すぐに気付いてくださってありがとうございました！
